ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/tasks/DetermineCommonAncestorTask.java

```diff
@@ -142,8 +148,13 @@ static int calculateCount(final double range, final int skipInterval) {
 
   private CompletableFuture<Void> processHeaders(
       final AbstractPeerTask.PeerTaskResult<List<BlockHeader>> headersResult) {
+    final CompletableFuture<Void> processingResult = CompletableFuture.completedFuture(null);
```

Comment:
optional nit: do we need to hold a variable for this?  we can call `CompletableFuture.completedFuture(null)` in both returns and never use it more than once.  Feels like too much normalization.

Reply:
Yeah, I think it does end up being a bit confusing.  I'll clean this up.

Reply:
Fixed